objects/src/transaction/created_notes.rs

```diff
@@ -101,3 +101,15 @@ pub fn generate_created_notes_stub_commitment(notes: &[NoteStub]) -> Digest {
 
     Hasher::hash_elements(&elements)
 }
+
+impl From<NoteStub> for NoteEnvelope {
+    fn from(note_stub: NoteStub) -> Self {
+        Self::new(note_stub.hash(), *note_stub.metadata())
```

Comment:
`NoteStub` contains a `NoteEnvelope` field.  Maybe we should use this instead of accessing `.has()` and `.metadata()`

Reply:
Yep, I think this should work.